.idea/runConfigurations/Unittests__Partial.xml

```diff
@@ -14,7 +14,7 @@
     <option name="ADD_SOURCE_ROOTS" value="true" />
     <EXTENSION ID="PythonCoverageRunConfigurationExtension" runner="coverage.py" />
     <option name="SCRIPT_NAME" value="unittest" />
-    <option name="PARAMETERS" value="discover -s tests -p &quot;*questionnaire*&quot;" />
+    <option name="PARAMETERS" value="discover -s tests -p &quot;*update_ehr_status*&quot;" />
```

Comment:
May be a "don't care" but is this file a candidate for .gitignore?

Reply:
Sorry, forgot to revert that one before adding it to the commit.